packages/clerk-js/src/core/errors.ts

```diff
@@ -113,3 +113,9 @@ export function clerkInvalidRoutingStrategy(strategy?: string): never {
 export function clerkUnsupportedReloadMethod(className: string): never {
   throw new Error(`${errorPrefix} Calling ${className}.reload is not currently supported. Please contact support.`);
 }
+
+export function clerkMissingWebAuthnPublicKeyOptions(name: 'create' | 'get'): never {
+  throw new Error(
+    `${errorPrefix} Missing publicKey. When calling 'navigator.credentials.${name}()' it is required to pass a publicKey object.`,
+  );
```

Comment:
❓  Maybe we should be more descriptive of what the `publicKey` refers to?

Reply:
Where this function is used @panteliselef wrote `// This should never occur, just a fail-safe` so I'm assuming this is for handling edge cases. If this would be surface to the user a CTA to contact us might be helpful since we ourselves don't think they'll ever get into that state